src/layout/component/code-demo.js

```diff
@@ -41,7 +43,7 @@ const CodeDemo = ({ iframe, iframeFullwidth, height = null, code, _body, _ID, _r
 								<div className="code-demo__example__code">
 									<div className="tabs">
 										<nav className="tabs-nav">
-											<ul className="au-link-list au-link-list--inline ">
+											<ul className="au-link-list au-link-list--inline" role="tablist">
 												{
```

Comment:
If we added `role="presentation"` to the `<li>`'s, it would ignore its list item state.

The other option is to replace the `<ul role="tablist"> <li> <a role="tab">` with 

```
<div role="tablist">
<a role="tab">
<a role="tab">
</div>
```

Reply:
see here for more info:

https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html

Reply:
Thanks @sukhrajghuman for the tip, I have updated the code.